python-lib/dku_neo4j/neo4j_handle.py

```diff
@@ -518,6 +524,9 @@ def __init__(
         if self.relationship_id_column:
             self.used_columns.append(self.relationship_id_column)
 
+        self.na_values = na_values
+        self.keep_default_na = keep_default_na
```

Comment:
to be consistent with the other expert parameters, make sure to only use these 2 params when expert mode is activated
```suggestion
        self.na_values = na_values if expert_mode else None
        self.keep_default_na = keep_default_na if expert_mode else True
```